Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -2258,6 +2269,18 @@ confirmingLottieView push state =
           ]
     ]
 
-genderBanner :: forall w . (Action -> Effect Unit) -> HomeScreenState -> PrestoDOM (Effect Unit) w
-genderBanner push state =
-  Banner.view (push <<< GenderBannerModal) (genderBannerConfig state)
\ No newline at end of file
+checkAppVersion :: forall action. (action -> Effect Unit) -> action -> HomeScreenState -> Flow GlobalState Unit
+checkAppVersion push action state = do
+  if os == "IOS" then do 
+      apires <- getAppStoreVersion (getValueFromConfig "appId")
+      versionName <- pure $ unsafePerformEffect getVersionName
```

Comment:
use `liftFlow` instead of `unsafePerformEffect`

Reply:
Done